Libraries/Components/WebView/WebView.android.js

```diff
@@ -153,6 +153,12 @@ var WebView = React.createClass({
      * start playing. The default value is `false`.
      */
     mediaPlaybackRequiresUserAction: PropTypes.bool,
+    
+    /**
+     * Used on Android(5.0+), controls whether Mixed content mode is enabled or not
+     * @platform android
+     */
+    enableMixedContentMode: PropTypes.bool,
```

Comment:
can you rename this to mixedContentModeEnabled actually? other props in this component are named xxxEnabled